addons/website_profile/views/website_profile.xml

```diff
@@ -630,16 +630,22 @@
 
         <div t-if="not validation_email_sent and not is_public_user and user.karma == 0" t-att-class="send_alert_classes" role="alert">
             <button type="button" class="close validation_email_close" data-dismiss="alert" aria-label="Close">×</button>
-            It appears your email has not been verified.<br/>
-            <a class="send_validation_email alert-link" href="#" t-att-data-redirect_url="redirect_url">
-                <span t-esc="send_validation_email_message"/>
-            </a>
+            Your Account has not yet been verified.<br/>
+            Click <a class="send_validation_email alert-link" href="#" t-att-data-redirect_url="redirect_url"><u>here</u></a> to receive a verification email<span t-esc="send_validation_email_message"/>
```

Comment:
I would add a space before the `<span/>`, also is the span needed ? (no need to create a HTML element, you can use `<t/>`)
```suggestion
            Click <a class="send_validation_email alert-link" href="#" t-att-data-redirect_url="redirect_url"><u>here</u></a> to receive a verification email <t t-esc="send_validation_email_message"/>!
```

Reply:
I changed the span to t. If no space it added, the sentence will be 
"Click here to receive a verification email !"
with a space before the exclamation mark.